src/charm.py

```diff
@@ -79,7 +76,7 @@ def _on_fluentbit_relation_created(self, event):
 
     def _configure_fluentbit(self):
         logger.debug("## Configuring fluentbit")
-        cfg = list()
```

Comment:
I'm the one who advocates for the `list()` or `dict()` or whatever definition. I'm curious about the reason to change it :)

Reply:
my concern is more related to readability. We have had people in the team with some sort of dyslexia, so, `[]` or `{}` was difficult to read sometimes, that's why we always have preffered `list()` over `[]` for example

Reply:
but I do think you have a good argument. Let's move on